src/main/java/utility/commandChecker.java

```diff
@@ -0,0 +1,114 @@
+package utility;
+
+public class commandChecker {
```

Comment:
Class names should be in PascalCase. [See here](https://se-education.org/guides/conventions/java/basic.html#naming)